packages/zero-client/src/client/options.ts

```diff
@@ -128,7 +128,7 @@ export interface ZeroOptions<
    * await z.mutate.issues.close({id: 'issue-123'}).client;
    * ```
    */
-  mutators?: MD | undefined;
+  mutators?: MD extends CustomMutatorDefs ? MD : AnyMutatorRegistry | undefined;
```

Comment:
I'm confused why the conditional type here and not `MD | AnyMutatorRegistry | undefined`

Reply:
If MD is a MutatorRegistry we do not care about the real shape here. It is not important for type safety.